server/api/jellyfin.ts

```diff
@@ -93,9 +93,7 @@ export interface JellyfinLibraryItemExtended extends JellyfinLibraryItem {
 }
 
 class JellyfinAPI extends ExternalAPI {
-  private authToken?: string;
   private userId?: string;
-  private jellyfinHost: string;
 
   constructor(jellyfinHost: string, authToken?: string, deviceId?: string) {
     let authHeaderVal: string;
```

Comment:
(not sure how to comment on the line below this)

We could remove the client, device, deviceId, version now that authorization this is handled using api token
https://gist.github.com/nielsvanvelzen/ea047d9028f676185832e51ffaf12a6f#examples

Reply:
Oh wait. You're right. When users login, that will use these auth headers too. My bad, didn't think about that